tests/suites/main_test.function

```diff
@@ -24,6 +24,7 @@
 #endif
 
 #include "mbedtls/build_info.h"
+#include "common.h"
```

Comment:
There are things in common.h like MBEDTLS_STATIC_TESTABLE and MBEDTLS_ALLOW_PRIVATE_ACCESS that don't really make sense to be included here thus I would just add "extern void (*mbedtls_test_hook_test_fail)( const char * test, int line, const char * file );" in this file before to use it: line 241.